### PR TITLE
Resolved issues 231, 233

### PR DESCRIPTION
**Pull request by potvik**

### CODE REVIEW

File: src/components/TransactionPage.vue [hunk 1]
```diff
@@ -36,21 +36,16 @@
...
                   Value
                 </td>
-                <td>
-                  <vue-json-pretty :data="transaction.msg" />
+                <td v-if="isStaking && transaction.type === 'EditValidator'">
+                  -
                 </td>
+                <td v-else>{{ transaction.value | amount }}</td>
               </tr>
+
```
Comment (denniswon) [inline src/components/TransactionPage.vue:48]: we are already showing the correct value in the staking txns list views right?

Comment (potvik) [reply src/components/TransactionPage.vue:48]: v-if="isStaking && transaction.type === 'EditValidator"

It is from issue - For Edit Validator Transaction, the 'Value' row should have a blank '-'

File: src/components/TransactionPage.vue [hunk 2]
```diff
@@ -130,24 +125,28 @@
...
+                <td class="td-title">Delegator Address</td>
+                <td>
+                  <router-link
+                    v-if="transaction.delegator"
+                    :to="'/address/' + transaction.delegator"
+                    >{{ transaction.delegator }}</router-link
+                  >
+                  <span v-else>-</span>
+                </td>
+              </tr>
```
[the remark this answers is not in the document]
Comment (potvik) [reply src/components/TransactionPage.vue:149]: It is github view
please look at raw version https://github.com/harmony-one/harmony-dashboard/blob/57bd596b224335d9c865abc6b27389dbcfa3530d/src/components/TransactionPage.vue

File: src/components/TransactionPage.vue (hunk 1 above)
Comment (denniswon) [reply src/components/TransactionPage.vue:48]: I mean on StakingTransactionsTable

Comment (denniswon) [reply src/components/TransactionPage.vue:48]: just checked in the code. we already do show value as tx.msg.amount. cool. thx

**Pull request merged**